bouncer/shared/lp_api_test.ts

```diff
@@ -47,7 +47,9 @@ async function provideLiquidityAndTestAssetBalances() {
   let ethBalance = 0;
   do {
     const balances = await lpApiRpc(`lp_asset_balances`, []);
-    ethBalance = balances.Eth;
+    ethBalance = parseInt(
```

Comment:
Like it seemed this test has never tested anything, ethBalance was undefined and undefined < a number is false :D. Thanks JavaScript :D.